mkdocs/technical/plugins/hooks.md

```diff
@@ -779,5 +780,151 @@ Not all the hooks have been documented yet. ( [help needed! &#128568;](https://d
             - [Plugin object](https://github.com/cheshire-cat-ai/core/blob/main/core/cat/mad_hatter/plugin.py#L25)
 
     3. **Input arguments**  
+        This hook has no input arguments.
+
+        ??? example
+
+            ```python
+            from cat.mad_hatter.decorators import plugin
+            from pydantic import BaseModel, Field
+
+            # define your plugin settings model
+            class MySettings(BaseModel):
+            text_field: str = Field(
+                title="Your field title",
+                description="Your field description",
+                default="""default value""",
+                extra={"type": "Text"}
```

Comment:
If I'm picky just send me to hell :smile: what about setting real example values? we can just copy the C.A.T. plugin or invent new ones

Reply:
lol, I will change it, I took that plugin data, and changed them in general ones ahah, will take those and make a real usage example